java/test/jmri/jmrix/loconet/LnOpsModeProgrammerTest.java

```diff
@@ -466,6 +466,78 @@ public void testOpsReadLocoNetMode() throws ProgrammerException {
 
      }
 
+     @Test
+     public void testOpsReadLocoNetModeLACKRejected() throws ProgrammerException {
+        // allow transponding
+        sm.setTranspondingAvailable(false);
+
+        LnOpsModeProgrammer lnopsmodeprogrammer = new LnOpsModeProgrammer(memo, 4, true);
+
+        lnopsmodeprogrammer.setMode(LnProgrammerManager.LOCONETOPSBOARD);
+        lnopsmodeprogrammer.readCV("12", pl);
+
+        // should have written
+        Assert.assertEquals("one message sent", 1, lnis.outbound.size());
+        Assert.assertEquals("No programming reply", 0, pl.getRcvdInvoked());
+
+        Assert.assertEquals("sent", "EF 0E 7C 2F 00 00 04 00 00 0B 00 7F 7F 00", lnis.outbound.get(0).toString());
+
+        // check echo of sent message has no effect
+        LocoNetMessage m = lnis.outbound.get(0);
```

Comment:
Stylistic comment: you could have done this _before_ the previous assert, and checked the expected value against the actual value `m.toString()`.  _No change required!_

Reply:
@devel-bobm  Again i just copied the existing testOpsReadLoconetMode() test case as a starting point.  I am not experienced in this area and don't feel comfortable commenting on test case patterns and styles.